### ISSUE TITLE
Unable to Connect to the New Git Repo

### ISSUE THREAD

**Issue by zhfe99** [railwaycat/emacs-mac-port#88]
I tried many times to directly connect Yamamoto Mitsuharu's repo via:
git clone http://www.math.s.chiba-u.ac.jp/~mituharu/emacs-mac.git/

But it seems my git cannot clone the repo successfully.

**Comment by pitkali** [comment]
Worked for me this morning with git 2.4.0.

**Comment by zhfe99** [comment]
It works now.

I found the connection was a little slow.

**Comment by xuchunyang** [comment]
Unable to connect right now, I'm using https://github.com/kfowler/emacs-mac instead.

**Comment by rodlogic** [comment]
I have been trying since Friday night. The connection is incredibly slow. After literally hours of downloading I get the following:

```
Cloning into '/Library/Caches/Homebrew/emacs-mac--git'...
error: Unable to get pack file http://www.math.s.chiba-u.ac.jp/~mituharu/emacs-mac.git/objects/pack/pack-c3a8f23a98140df3eed0deeefe89a031c97e0eae.pack
transfer closed with 135609843 bytes remaining to read
error: Unable to find ea0e13ddab7c55bb27e8e81c6344d76d8c5c3db8 under http://www.math.s.chiba-u.ac.jp/~mituharu/emacs-mac.git
Cannot obtain needed object ea0e13ddab7c55bb27e8e81c6344d76d8c5c3db8
while processing commit 029d156140c2faca3c2c6b256f30f9593cd605c4.
error: fetch failed.
Error: Failed to download resource "emacs-mac"
Failure while executing: git clone http://www.math.s.chiba-u.ac.jp/~mituharu/emacs-mac.git /Library/Caches/Homebrew/emacs-mac--git
```

**Comment by railwaycat** [comment]
opened [new issue](https://github.com/railwaycat/homebrew-emacsmacport/issues/28) on homebrew formula side to tracking repo switch, which would speed up the cloning and update.
